src/DataShare/DataShare.Test/ScenarioTests/AccountTests.ps1

```diff
@@ -0,0 +1,46 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Full Account CRUD cycle
+#>
+function Test-AccountCrud
+{
+    $AccountName = getAssetName
```

Comment:
@gaurangisaxena I would recommend refactoring your tests to the following structure:

```powershell
# Define variables
try
{
    # Create resources necessary for this test
    # Verify behavior using assertions
}
finally
{
    # Delete resources created during this test
}
```

This ensures that during test recording, if any errors are thrown, the resources created during the test are deleted so there's no impacting the next test recording.